generate-download-stats/package.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
I would rename this file and then description to make it clear it's for homebrew